test/ReadsAlignmentUtils_server_test.py

```diff
@@ -93,9 +93,11 @@ def tearDownClass(cls):
         if hasattr(cls, 'nodes_to_delete'):
             for node in cls.nodes_to_delete:
                 cls.delete_shock_node(node)
+        #TODO:Fix handle delete. Error is handle not owned by user
         if hasattr(cls, 'handles_to_delete'):
-            cls.hs.delete_handles(cls.hs.ids_to_handles(cls.handles_to_delete))
-            print('Deleted handles ' + str(cls.handles_to_delete))
+            pass
+            #cls.hs.delete_handles(cls.hs.ids_to_handles(cls.handles_to_delete))
```

Comment:
change it to `cls.hs.delete_handles(cls.hs.hids_to_handles(cls.handles_to_delete))`

Reply:
Thanks! This wasted a lot of my time.